web/tls_config.go

```diff
@@ -226,6 +227,24 @@ func ListenAndServe(server *http.Server, flags *FlagConfig, logger log.Logger) e
 	return ServeMultiple(listeners, server, flags, logger)
 }
 
+// compact replaces consecutive runs of equal elements with a single copy.
```

Comment:
Hmm, shouldn't we fully unique the slice rather than just consecutive duplicates?

Reply:
yes, maybe converting to a set is better here. The question is more, does it makes sense to dedup on this part of the code?